src/main/java/genji/Storage.java

```diff
@@ -0,0 +1,81 @@
+package genji;
+
+import genji.task.Task;
+import genji.task.TaskList;
+import genji.task.ToDo;
+import genji.task.Deadline;
+import genji.task.Event;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Scanner;
+
+public class Storage {
+   private String filePath;
+
+   public Storage(String filePath) {
+       this.filePath = filePath;
+   }
+
+    public TaskList loadList() {
```

Comment:
For **public functions** for classes remember to add JavaDocs.

**For example:**
```java
/**
 * <Function Description>.
 *
 * @return <What does it return>.
 */
public TaskList loadList()
    // Function 
}
```